### ISSUE TITLE
Error when adding devServer type

### ISSUE THREAD

**Issue by PierBJX** [webpack/webpack-dev-server#4213]
### Bug report

With typescript, when adding types for devServer, I have an error when I try to define the `bypass` property.

```
    "typescript": "4.4.4",
    "webpack": "^5.66.0",
    "webpack-cli": "^4.9.1",
    "webpack-dev-server": "^4.7.3",
```
### Actual Behavior

It throws this error:
` 'bypass' does not exist in type 'Options'.`

### How Do We Reproduce?

```
/// <reference path="../../node_modules/webpack-dev-server/types/lib/Server.d.ts"/>

const test: Configuration['devServer'] = {
  proxy: {
    '/api': {
      target: 'http://localhost:3000',
      bypass: function (req, res, proxyOptions) {
        if (req.headers.accept.indexOf('html') !== -1) {
          console.log('Skipping proxy for browser request.');
          return '/index.html';
        }
      },
    },
  },
}
```

**Comment by alexander-akait** [comment]
Feel free to fix types

**Comment by PierBJX** [comment]
It seems fixed with the last release `4.7.4`

https://github.com/webpack/webpack-dev-server/compare/v4.7.3...v4.7.4
